File: 70/climbing-stairs.java
class Solution {
    public int climbStairs(int n) {
        int [] table = new int[n+1];
        table[0] = 1;
        table[1] = 1;
        for(int i=2; i<=n; i++){
            table[i] = table[i-1] + table[i-2];
        }
        return table[n];
    }
}

File: 70/climbing-stairs.py
class Solution:
    def climbStairs(self, n: int) -> int:
        f = [0]*46
        f[0] = 1
        f[1] = 1
        for i in range(2,n+1):
            f[i] = f[i-1] + f[i-2] 
        return f[n]
